kotlin/src/test/TestMethods.kt

```diff
@@ -70,14 +68,14 @@ class TestMethods {
         )
     }
 
-    private fun slowQuery(): WriteQuery {
-        return WriteQuery(
-            "system__activity",
-            "dashboard",
-            arrayOf("dashboard.id", "dashboard.title", "dashboard.count"),
-            limit = "5000"
-        )
-    }
+//    private fun slowQuery(): WriteQuery {
```

Comment:
Do you want to add this back in or remove it entirely?

Reply:
Probably remove it entirely. I have a hard time letting go of code!